xml/operations-compute-gpu_passthrough.xml

```diff
@@ -24,14 +24,28 @@
   <para>
     The following steps are necessary to leverage PCI passthrough on a &cloud;
     &productnumber; &compnode;: preparing the &compnode;, preparing &o_comp;
-    via the input model updates and &o_img;.
+    via the input model updates and &o_img;. Ensure you follow the
+    below procedures in sequence:
   </para>
   <orderedlist>
-  <listitem>
-      <para>
-        <emphasis role="bold">Preparing the &compnode;</emphasis>
-      </para>
-      <procedure>
+   <listitem>
+    <para>
+     <xref linkend="clm-prepare-comp-node"/>
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <xref linkend="clm-prep-comp-input-model"/>
+    </para>
+  </listitem>
+   <listitem>
+    <para>
+     <xref linkend="clm-create-flavor"/>
+    </para>
+  </listitem>
+ </orderedlist>
+  <procedure xml:id="clm-prepare-comp-node">
```

Comment:
Debating turning this into a section too - @sknorr ?

Reply:
Sounds reasonable. Otherwise, lgtm.